### ISSUE TITLE
[RE1] Stuck in Piano Room unless you use debug menu

### ISSUE THREAD

**Issue by sakurayosei** [IntelOrca/biorand#790]
### Version

3.2.2

### Seed

R710-K862-ZNHH40M5Y9Y00H3N9KMUM494ZZZZZZZZZZZZZZZZZZZZZZZZZ

### Scenario

RE 1 (Jill)

### Player Character

Enrico

### What happened?

I know this room has been talked about before and fixed(?) recently yet I think i still encountered it. Grabbed the item behind bookcase in piano bar. doing so prompted the cutscene where the wall slid down to the alcove with emblem statue. camera was stuck there and i could not see where i was. Reloading the room via debug menu fixed it but thought i should let you know its still happening.

**Comment by IntelOrca** [comment]
How do I reproduce this? Are there some reliable steps from the start or can you upload a saved game?

The problem before was that the quantity of the music notes needed to be 1, not 0.

**Comment by sakurayosei** [comment]
so i tried to back and reproduce it myself. but i updated to 3.2.3 after this run and when i put this seed in things were different? i.e. where doors led, enemies, etc. i used debug menu to get back to the bar and picked up item on bookcase but it was normal this time. it didnt trigger the cutscene for the wall closing down again.

i'm trying to think if there was anything else i could have done that would cause the game to do that. i'll upload my save regardless. 
[SakuraYosei_BarSaveData.zip](https://github.com/IntelOrca/biorand/files/13952671/SakuraYosei_BarSaveData.zip)

**Comment by IntelOrca** [comment]
Yeah if you have to use the same version that you originally used.

**Comment by Matakomi** [comment]
How do I enable this debug menu? Can I get items from it? I'm trying to mix the V-Jolt, but when I try to combine two **UMB No. 13** It just drains out one bottle and the other stay as **UMB No. 13**

**Comment by IntelOrca** [comment]
F5, but you might need to enable it first by pressing F1 on the CR launcher.
